web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -1,13 +1,13 @@
-<div layout-padding layout="column" class="map-page"> <!--layout-align="center center"> -->
+<div layout-padding layout="column" class="map-page">
 
-  <div *ngIf="targetEntity && !targetEntity.hasDocs" class="map-container2">
+  <div *ngIf="targetEntity && !targetEntity.hasDocs" class="map-container">
     <p>Before configuring this mapping you must ingest source documents</p>
   </div>
 
-  <div *ngIf="targetEntity && targetEntity.hasDocs" class="map-container2">
+  <div *ngIf="targetEntity && targetEntity.hasDocs" class="map-container">
 
     <div id="source">
-      <div id="src-heading" #srcheading>
+      <div id="source-heading" #srcheading>
```

Comment:
#source-heading instead of #srcheading?

Reply:
@ayuwono, removed #srcheading, not used.